http_server/response.pony

```diff
@@ -1,6 +1,7 @@
-use "valbytes"
 use "format"
 
+use "valbytes"
+
 interface val Response is ByteSeqIter
```

Comment:
This is unrelated at this point, no new uses were added so I think this should be rolled back